lib/matplotlib/font_manager.py

```diff
@@ -1308,6 +1309,100 @@ def findfont(self, prop, fontext='ttf', directory=None,
             prop, fontext, directory, fallback_to_default, rebuild_if_missing,
             rc_params)
 
+    def find_fontsprop(self, prop, fontext='ttf', directory=None,
```

Comment:
We talked on the phone about renaming this.  The should invoke that is is going at a) find multiple fonts b) that the order of the returned fonts matters.

Reply:
Some suggestions:
1. `find_fonts`
2. `find_orderedfonts`
3. `find_fontsdict`
4. `find_fontorder`
5. `find_fontset`
6. `find_fontfamilies`
7. `find_orderedfamily`
8. `find_familyordered`
9. `find_families`
10. `find_fontfamilies`